<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabEtikette</class>
 <widget class="QWidget" name="TabEtikette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_27">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_FLabelAuswahl">
         <property name="title">
          <string>Etikette</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_24">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="label_FLabelAuswahl">
            <property name="text">
             <string>Auswahl aus Anhänge im Rezept (svg)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ComboBox" name="cbAuswahl"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>PDF Ausgabe</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_28">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="label_77">
            <property name="text">
             <string>Breite Label (mm)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_BreiteLabel">
            <property name="maximum">
             <number>210</number>
            </property>
            <property name="value">
             <number>185</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_49">
            <property name="text">
             <string>Anzahl Labels</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_AnzahlLabels">
            <property name="maximum">
             <number>9999999</number>
            </property>
            <property name="value">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_76">
            <property name="text">
             <string>Abstand zwischen den Labels (mm)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_AbstandLabel"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_21">
            <property name="title">
             <string>Seitenränder</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_37">
             <property name="spacing">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="label_72">
               <property name="text">
                <string>Oben (mm)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_FLabel_RandOben">
               <property name="maximum">
                <number>297</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_73">
               <property name="text">
                <string>Links (mm)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_FLabel_RandLinks">
               <property name="maximum">
                <number>210</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_74">
               <property name="text">
                <string>Rechts (mm)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_FLabel_RandRechts">
               <property name="maximum">
                <number>210</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_75">
               <property name="text">
                <string>Unten (mm)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_FLabel_RandUnten">
               <property name="maximum">
                <number>290</number>
               </property>
               <property name="value">
                <number>15</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoeschen">
         <property name="text">
          <string>Etikette löschen</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <layout class="QVBoxLayout" name="verticalLayout_23">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <widget class="SvgView" name="viewSvg"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QCheckBox" name="cbEditMode">
               <property name="text">
                <string>Editiermodus</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbTagsErsetzen">
               <property name="text">
                <string>Tags ersetzen</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnSaveTemplate">
               <property name="text">
                <string>Speichern</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QPlainTextEdit" name="tbTemplate">
           <property name="lineWrapMode">
            <enum>QPlainTextEdit::NoWrap</enum>
           </property>
          </widget>
          <widget class="QTreeWidget" name="treeViewTemplateTags">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <column>
            <property name="text">
             <string notr="true">Tag</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string notr="true">Wert</string>
            </property>
           </column>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnToPdf">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>PDF speichern</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/combobox.h</header>
  </customwidget>
  <customwidget>
   <class>SvgView</class>
   <extends>QGraphicsView</extends>
   <header>widgets/svgview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbAuswahl</tabstop>
  <tabstop>spinBox_BreiteLabel</tabstop>
  <tabstop>spinBox_AnzahlLabels</tabstop>
  <tabstop>spinBox_AbstandLabel</tabstop>
  <tabstop>spinBox_FLabel_RandOben</tabstop>
  <tabstop>spinBox_FLabel_RandLinks</tabstop>
  <tabstop>spinBox_FLabel_RandRechts</tabstop>
  <tabstop>spinBox_FLabel_RandUnten</tabstop>
  <tabstop>cbEditMode</tabstop>
  <tabstop>cbTagsErsetzen</tabstop>
  <tabstop>btnSaveTemplate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
